### PR TITLE
[INLONG-3856][Manager] Add heartbeat common report handler

### LINKED ISSUE #3856: [Feature][Manager] Add heartbeat report handler
### Description

Add a heartbeat report handler to support the other components (such as Agent, DataProxy, etc.) to report their heartbeat.

### Use case

Add a heartbeat report handler.
image
image

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by baomingyu**
### Title Name: [INLONG-3856][Manager] Add a heartbeat report handler

Fixes #3856

### CODE REVIEW

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/beatheart/InlongBeatHeartInfoResponse.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+package org.apache.inlong.manager.common.pojo.beatheart;
+
+import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class InlongBeatHeartInfoResponse {
```
Comment (EMsnap) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/beatheart/InlongBeatHeartInfoResponse.java:26]: heartbeat

Comment (baomingyu) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/beatheart/InlongBeatHeartInfoResponse.java]: done

**Comment by healchow** [comment]
Let me check later.

File: inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/openapi/HeartBeatControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                + "\"streamMetrics\":[{\"streamId\":\"stream1\"}]}]");
+        componentStaticInfo.setInlongGroupId("group1");
+        componentStaticInfos.add(componentStaticInfo);
+        request.setComponentStaticInfos(componentStaticInfos);
+
+        Assert.assertEquals("Success", heartbeatService.reportHeartbeatInfo(request));
+    }
+
+    @Test
+    public void tesQuerytHeartBeat() throws Exception {
```
Comment (dockerzhang) [inline inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/openapi/HeartBeatControllerTest.java:132]: typo

File: inlong-manager/manager-dao/src/main/resources/generatorConfig.xml [hunk 3]
```diff
@@ -239,6 +239,24 @@
                enableDeleteByPrimaryKey="true" enableInsert="true"
                enableCountByExample="false" enableDeleteByExample="false"
                enableSelectByExample="false" enableUpdateByExample="false"/>-->
+        <table tableName="inlong_heartbeat" domainObjectName="InlongHeartbeatEntity"
+            enableSelectByPrimaryKey="true"
+            enableUpdateByPrimaryKey="true"
+            enableDeleteByPrimaryKey="true" enableInsert="true"
+            enableCountByExample="false" enableDeleteByExample="false"
+            enableSelectByExample="false" enableUpdateByExample="false"/>
+<!--        <table tableName="inlong_stream_status_info" domainObjectName="InlongStreamStatusEntity"-->
```
Comment (dockerzhang) [inline inlong-manager/manager-dao/src/main/resources/generatorConfig.xml:248]: remove the annotated lines.

File: inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/openapi/HeartBeatControllerTest.java (hunk 2 above)
Comment (baomingyu) [reply inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/openapi/HeartBeatControllerTest.java]: done

File: inlong-manager/manager-dao/src/main/resources/generatorConfig.xml (hunk 3 above)
Comment (baomingyu) [reply inlong-manager/manager-dao/src/main/resources/generatorConfig.xml]: done

File: inlong-manager/manager-web/sql/apache_inlong_manager.sql [hunk 4]
```diff
@@ -1108,5 +1108,67 @@ CREATE TABLE `stream_config_log`
...
+    `metric_heartbeat`        text         NOT NULL COMMENT 'component metric info',
+    `report_time`             timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'report time',
+    `modify_time`             timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'Modify time',
+    `create_time`             timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'create time',
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `index_beat_heart` (`component`, `instance`),
+    KEY `index_report_time` (`report_time`)
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8 COMMENT ='inlong heartbeat';
+
```
Comment (gong) [inline inlong-manager/manager-web/sql/apache_inlong_manager.sql:1129]: `manager-test` also need update sql script

Comment (baomingyu) [reply inlong-manager/manager-web/sql/apache_inlong_manager.sql:1129]: done

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**